Release/public-v2: Bugfix for lmodsetup files on noaacloud

gspetro-NOAA opened the pull request:
## DESCRIPTION OF CHANGES: 
On NOAA Cloud's GCP and Azure systems, use of the current lmod-setup files produces a problem finding the Lmod path. Removing the noaacloud stanza from lmod-setup fixes the problem.
Also updates hash of regional_workflow to HEAD of release/public-v2.

## TESTS CONDUCTED: 
Built SRW App successfully from the release branch with devbuild.sh method on AWS, GCP, Azure. 
Bruce Kropp also confirmed the builds. 

## DOCUMENTATION:
N/A

## CONTRIBUTORS (optional): 
@natalie-perlin @EdwardSnyder-NOAA @ulmononian Bruce Kropp

gspetro committed:
fixed ref links

gspetro committed:
finished Intro

gspetro committed:
finish Components & Intro edits

gspetro committed:
edited Rocoto workflow section of Quickstart

gspetro committed:
added minor hpc submodule commits

gspetro committed:
Updates to Rocoto Workflow in Quick Start

gspetro committed:
add to HPC-stack intro

gspetro committed:
submodule updates

gspetro committed:
added submodule docs edits

gspetro committed:
hpc-stack updates & formatting fixes

gspetro committed:
hpc-stack intro edits

gspetro committed:
bibtex attempted fix

gspetro committed:
add hpc-stack module edits

gspetro committed:
update sphinxcontrib version

gspetro committed:
add .readthedocs.yaml file

gspetro committed:
update .readthedocs.yaml file

gspetro committed:
update .readthedocs.yaml file

gspetro committed:
update conf.py

gspetro committed:
updates .readthedocs.yaml with submodules

gspetro committed:
updates .readthedocs.yaml with submodules

gspetro committed:
submodule updates

gspetro committed:
submodule updates

gspetro committed:
minor Intro edits

gspetro committed:
minor Intro edits

gspetro committed:
minor Intro edits

gspetro committed:
submodule updates

gspetro committed:
fixed typos in QS

gspetro committed:
QS updates

gspetro committed:
QS updates

gspetro committed:
QS updates

gspetro committed:
updates to InputOutput and QS

gspetro committed:
fix I/O doc typos

gspetro committed:
pull updates to hpc-stack docs

gspetro committed:
pull updates to hpc-stack docs

gspetro committed:
fix table wrapping

gspetro-NOAA committed:
Merge branch 'ufs-community:develop' into develop

gspetro committed:
updates to QS for cloud

gspetro committed:
Merge branch 'develop' of github.com:gspetro-NOAA/ufs-srweather-app into develop

gspetro committed:
fix QS export statements

gspetro committed:
fix QS export statements

gspetro committed:
QS edits on bind, config

gspetro committed:
add bullet points to notes

gspetro committed:
running without rocoto

gspetro committed:
add HPC-Stack submodule w/docs

gspetro committed:
split QS into container/non-container approaches

gspetro committed:
added filepath changes for running in container on Orion, et al.

gspetro committed:
edits to overview and container QS

gspetro committed:
moved CodeReposAndDirs.rst info to the Introduction & deleted file

gspetro committed:
continued edits to SRWAppOverview

gspetro committed:
combine overview w/non-container docs

gspetro committed:
finish merging non-container guide & SRWOverview, rename/remove files, update FAQ

gspetro committed:
minor edits for Intro & QS

gspetro committed:
updates to BuildRun doc through 3.8.1

gspetro committed:
edits to Build/Run and Components

gspetro committed:
remove .gitignore

gspetro committed:
fix Ch 3 title, 4 supported platform levels note

gspetro committed:
fix typos, add term links

gspetro committed:
other minor fixes/suggestions implemented

gspetro committed:
updated Intro based on feedback; changed SRW to SRW App throughout

gspetro committed:
update comment to Intro citation

gspetro-NOAA committed:
Merge branch 'develop' into develop

gspetro committed:
add user-defined vertical levels to future work

gspetro committed:
Merge branch 'develop' of github.com:gspetro-NOAA/ufs-srweather-app into develop

gspetro-NOAA committed:
Merge branch 'ufs-community:develop' into develop

gspetro committed:
Add instructions for srw_common module load

gspetro committed:
Pull changes from upstream

gspetro committed:
fix typo

gspetro committed:
update Intro & BuildRunSRW based on Mark's feedback

gspetro committed:
minor intro updates

gspetro committed:
1st round of jwolff's edits

gspetro committed:
2nd round of jwolff updates

gspetro committed:
update QS intro

gspetro committed:
fix minor physics details

gspetro committed:
update citation and physics suite name

gspetro committed:
add compute node allocation info to QS

gspetro committed:
add authoritative hpc-stack docs to Intro

gspetro-NOAA committed:
Merge branch 'ufs-community:develop' into develop

gspetro committed:
edit config intro & platform environment sections

gspetro committed:
Merge remote-tracking branch 'origin/develop' into textonly/config

gspetro committed:
edit sections on cron & directory parameters, platform & parameters for running without a workflow manager

gspetro committed:
edit NCO, file-separator, filename params, add some METplus and model config params

gspetro committed:
ConfigWorkflow.rst revisions, added METplus to Components, grid info

gspetro committed:
add grid config details

gspetro committed:
changes to readme.md

gspetro committed:
RTD readme.md edits

gspetro-NOAA committed:
Merge branch 'ufs-community:develop' into develop

gspetro committed:
create MacOS install/build instructions

gspetro committed:
Merge branch 'develop' of github.com:gspetro-NOAA/ufs-srweather-app into develop

gspetro committed:
update task run and grid parameters

gspetro committed:
fixed file params & workflow task params

gspetro committed:
1st draft of ConfigParameters.inc

gspetro-NOAA committed:
Merge branch 'ufs-community:develop' into textonly/config

gspetro committed:
minor edits

gspetro committed:
Merge branch 'develop' into textonly/config

gspetro committed:
Merge branch 'textonly/config' of github.com:gspetro-NOAA/ufs-srweather-app into textonly/config

gspetro committed:
add stochastic physics var details

gspetro committed:
update FVCOM, thread affinity params

gspetro committed:
halo_blend, ens, crtm, custom post, subhourly updates

gspetro committed:
update HPC-Stack submodule/docs

gspetro committed:
Merge branch 'develop' into textonly/config

gspetro committed:
remove extra macinstall document

gspetro committed:
Rocoto WF tasks & params

gspetro committed:
workflow tasks/params, debug, verbose, pre-existing dir, predefined grid

gspetro committed:
move Stochastic physics to CCP section; write component edits

gspetro committed:
comp'l forecast, grid gen, NOMADS, user-staged files

gspetro committed:
METplus, model config & forecast params, separator

gspetro committed:
2nd draft complete

gspetro committed:
physics updates

gspetro committed:
remove MacInstall empty file

gspetro committed:
undo hpc-stack submodule update (save for separate PR)

gspetro committed:
undo hpc-stack install doc update (save for separate PR)

gspetro committed:
revert hpc-stack submodule update

gspetro committed:
revisions to SPP & LSM physics

gspetro committed:
minor edits

gspetro committed:
update comments in LAM Grid chapter

gspetro committed:
update LSM_SPP_EACH_STEP

gspetro committed:
revert LSM_SPP_EACH_STEP to original definition

gspetro committed:
combine config info into one doc instead of two

gspetro-NOAA committed:
Merge branch 'ufs-community:develop' into develop

gspetro-NOAA committed:
Merge branch 'ufs-community:develop' into textonly/vx

gspetro committed:
remove ConfigParameters.inc (contents now appear in ConfigWorkflow.rst)

gspetro committed:
update hpc-stack docs submodule

gspetro committed:
odds & ends

gspetro-NOAA committed:
Merge branch 'ufs-community:develop' into develop

gspetro committed:
add VX tables, config info, & Rocoto output tables

gspetro committed:
add module use/load statements, fix typos

gspetro-NOAA committed:
Merge branch 'ufs-community:develop' into textonly/vx

gspetro committed:
varied minor details

gspetro committed:
add workflow svg diagram

gspetro-NOAA committed:
Merge branch 'ufs-community:develop' into develop

gspetro committed:
edits to rocoto ch

gspetro-NOAA committed:
Merge branch 'ufs-community:develop' into develop

gspetro committed:
update citation & readme

gspetro committed:
adjust comment in Intro

gspetro committed:
updates to Rocoto chapter

gspetro committed:
fix minor formatting/wording issues

gspetro-NOAA committed:
Merge branch 'ufs-community:develop' into develop

gspetro committed:
updates to LAMgrid chapter

gspetro-NOAA committed:
Merge branch 'ufs-community:develop' into develop

gspetro-NOAA committed:
Merge branch 'ufs-community:develop' into textonly/other

gspetro committed:
LAM Grid edits

gspetro committed:
Merge branch 'textonly/other' of github.com:gspetro-NOAA/ufs-srweather-app into textonly/other

gspetro committed:
LAM ch: user-defined grid section

gspetro committed:
add UPP Product tables ch 6

gspetro committed:
I/O edits & glossary terms

gspetro committed:
I/O Pt2

gspetro committed:
I/O changes

gspetro-NOAA committed:
Merge branch 'ufs-community:develop' into develop

gspetro committed:
include updated images

gspetro committed:
update docs to reflect changes in PR #238

gspetro committed:
Graphics Ch-1st pass

gspetro committed:
minor updates to Graphics

gspetro committed:
minor updates to Graphics

gspetro committed:
edit ConfigNewPlatform sections 1-4

gspetro committed:
ConfigNewPlatform edits

gspetro-NOAA committed:
Merge branch 'ufs-community:release/public-v2' into release/public-v2

gspetro committed:
update to match head of develop

gspetro committed:
resolve merge conflicts

gspetro committed:
I/O ch edits

gspetro committed:
I/O edits

gspetro committed:
more I/O edits

gspetro committed:
hpc-stack submodule updates

gspetro committed:
add HPC-Stack MacOs info

gspetro committed:
WE2E edits & tables

gspetro-NOAA committed:
Merge branch 'ufs-community:develop' into textonly/other

gspetro committed:
fix typo

gspetro committed:
Merge branch 'textonly/other' of github.com:gspetro-NOAA/ufs-srweather-app into textonly/other

gspetro committed:
minor grammar/typos

gspetro committed:
merge conflict resolution

gspetro-NOAA committed:
Merge branch 'ufs-community:develop' into develop

gspetro committed:
merge conflict resolution

gspetro committed:
fix grid name

gspetro committed:
remove resolved comments

gspetro committed:
add compact grids

gspetro committed:
file path updates & info for HPC-Stack

gspetro committed:
add SRW prereqs to Intro

gspetro committed:
change ConfigNewPlatform to a non-container quickstart

gspetro committed:
clean up non-container quickstart

gspetro committed:
update build options for non-container QS

gspetro committed:
update file paths & WE2E

gspetro committed:
minor fixes

gspetro committed:
update I/O & Gaea file paths

gspetro committed:
update error in non-container QS

gspetro committed:
add warning for users w/o Rocoto

gspetro committed:
add UPP Satellite Product instructions

gspetro committed:
Xlink for UPP satellite output info

gspetro committed:
clarify compiler prereqs

gspetro-NOAA committed:
Merge branch 'ufs-community:develop' into develop

gspetro committed:
update textonly/other to be even with develop

gspetro committed:
add list of fix files for release

gspetro committed:
Merge branch 'release/public-v2' of github.com:gspetro-NOAA/ufs-srweather-app into release/public-v2

gspetro committed:
update branch w/latest changes

gspetro committed:
fix typo

gspetro committed:
update input data file names

gspetro committed:
remove resolved comments

gspetro committed:
minor fix

gspetro-NOAA committed:
Include documentation for building/running SRW App on Mac (#240)

* updated docs

* added git submodule

* fix formatting

* added new submodule commits

* fixed ref links

* finished Intro

* finish Components & Intro edits

* edited Rocoto workflow section of Quickstart

* added minor hpc submodule commits

* Updates to Rocoto Workflow in Quick Start

* add to HPC-stack intro

* submodule updates

* added submodule docs edits

* hpc-stack updates & formatting fixes

* hpc-stack intro edits

* bibtex attempted fix

* add hpc-stack module edits

* update sphinxcontrib version

* add .readthedocs.yaml file

* update .readthedocs.yaml file

* update .readthedocs.yaml file

* update conf.py

* updates .readthedocs.yaml with submodules

* updates .readthedocs.yaml with submodules

* submodule updates

* submodule updates

* minor Intro edits

* minor Intro edits

* minor Intro edits

* submodule updates

* fixed typos in QS

* QS updates

* QS updates

* QS updates

* updates to InputOutput and QS

* fix I/O doc typos

* pull updates to hpc-stack docs

* pull updates to hpc-stack docs

* fix table wrapping

* updates to QS for cloud

* fix QS export statements

* fix QS export statements

* QS edits on bind, config

* add bullet points to notes

* running without rocoto

* add HPC-Stack submodule w/docs

* split QS into container/non-container approaches

* added filepath changes for running in container on Orion, et al.

* edits to overview and container QS

* moved CodeReposAndDirs.rst info to the Introduction & deleted file

* continued edits to SRWAppOverview

* combine overview w/non-container docs

* finish merging non-container guide & SRWOverview, rename/remove files, update FAQ

* minor edits for Intro & QS

* updates to BuildRun doc through 3.8.1

* edits to Build/Run and Components

* remove .gitignore

* fix Ch 3 title, 4 supported platform levels note

* fix typos, add term links

* other minor fixes/suggestions implemented

* updated Intro based on feedback; changed SRW to SRW App throughout

* update comment to Intro citation

* add user-defined vertical levels to future work

* Add instructions for srw_common module load

* fix typo

* update Intro & BuildRunSRW based on Mark's feedback

* minor intro updates

* 1st round of jwolff's edits

* 2nd round of jwolff updates

* update QS intro

* fix minor physics details

* update citation and physics suite name

* add compute node allocation info to QS

* add authoritative hpc-stack docs to Intro

* create MacOS install/build instructions

* add MacOS Build/Run instructions

* fix MacOS Build/Run details

* add MacOS info directly to Build/Run SRW chapter

* minor details

* minor edits

* update Include-HPCInstall with mac installation docs

* add note re: Terminal.app & bash shell in MacOS section

* remove MacInstall file-contents added to BuildRunSRW

* update hpc-stack submodule to include mac installation info

* add MacOS config details

* add MacOS config & run details

* minor MacOS note

* mention need to verify software library version #'s

* update hpc-stack-mod

* align MacDetails section with PR #238 info

* remove gsed & alter related commands

* update hpc-stack submodule

* typos

* switch from env to module load

Co-authored-by: Will Mayfield <59745143+willmayfield@users.noreply.github.com>

* Update BuildRunSRW.rst

* update hpc-stack module docs & MacOS config.sh

* update machine file instructions

* updates to BuildRun chapter

* fix typo

Co-authored-by: gspetro <gillian.s.petro@gmail.com>
Co-authored-by: Will Mayfield <59745143+willmayfield@users.noreply.github.com>

gspetro-NOAA committed:
Merge branch 'ufs-community:release/public-v2' into release/public-v2

gspetro committed:
update CCPP version typo

gspetro committed:
Merge branch 'release/public-v2' of github.com:gspetro-NOAA/ufs-srweather-app into release/public-v2

gsketefian committed:
Update documentation for CSV file containing WE2E test info (#278)

* Edits to documentation to match latest in code.

* Edits to documentation to match latest in code.

* Minor changes to documentation.

gspetro committed:
minor fixes

gspetro-NOAA committed:
Merge branch 'ufs-community:release/public-v2' into release/public-v2

gspetro committed:
fix typo, add comments

gspetro committed:
Merge branch 'release/public-v2' of github.com:gspetro-NOAA/ufs-srweather-app into release/public-v2

gspetro committed:
update UPP links

gspetro committed:
update links, add comments

gspetro committed:
add build/run alt text

gspetro-NOAA committed:
Merge branch 'ufs-community:release/public-v2' into release/public-v2

gspetro committed:
update HPC-Stack module to point to its srw release branch

gspetro committed:
update links

gspetro committed:
update QS container commands

natalie-perlin committed:
resolve conflicts w/PR #281

natalie-perlin committed:
merge changes from PR #240 to release-v2

gspetro committed:
QS updates

gspetro-NOAA committed:
Merge branch 'ufs-community:release/public-v2' into release/public-v2

gspetro committed:
I/O updates to wget commands

gspetro committed:
update HPSS file names

gspetro committed:
add links, remove comments

gspetro committed:
add links, remove comments

gspetro committed:
FAQ updates

gspetro committed:
ContribGuide updates

gspetro committed:
Rocoto ch updates

gspetro committed:
update wflow gen image

gspetro committed:
update LAM grid chapter

gspetro committed:
update docs links

gspetro committed:
update NC QS

gspetro committed:
glossary updates

gspetro committed:
add Stochastic Physics info

gspetro committed:
format valid values

gspetro committed:
edits for Config Params

gspetro committed:
final Config Param updates

gspetro committed:
Graphics edits

gspetro committed:
remove WCOSS info

gspetro committed:
Intro updates pt1

gspetro committed:
miscellaneous changes

gspetro committed:
Intro & QS updates

gspetro committed:
minor fixes

gspetro-NOAA committed:
Merge branch 'release/public-v2' into release/public-v2

gspetro committed:
remove noaacloud stanza in lmod-setup

gspetro committed:
revert doc changes

gspetro committed:
revert textonly changes

gspetro committed:
minor fix

gspetro committed:
minor fixes

gspetro committed:
minor fix

gspetro committed:
update readme

gspetro committed:
update RW hash to head of release branch

gspetro committed:
update Externals to reflect RW release branch

EdwardSnyder-NOAA commented:
@gspetro-NOAA - There are a number of files changing in the hpc-stack-mod directory. Did you intend to include these changes with this PR?

gspetro committed:
test

gspetro-NOAA commented:
> @gspetro-NOAA - There are a number of files changing in the hpc-stack-mod directory. Did you intend to include these changes with this PR?

@EdwardSnyder-NOAA Yes--I had updated the HPC-Stack submodule, so those changes came in. Seemed like a pain to undo them, and no harm done if they come in, since it will catch any minor doc updates (that's why I updated in the first place). The submodule isn't used for anything but the docs, so any changes in HPC-Stack functionality should have no direct impact.

gspetro-NOAA committed:
Merge branch 'release/public-v2' into bugfix/cloud-lmodsetup

EdwardSnyder-NOAA commented:
@gspetro-NOAA Okay, just wanted to make sure. I'll approve the PR.
